treeherder/log_parser/parsers.py

```diff
@@ -388,6 +388,7 @@ class ErrorParser(ParserBase):
         r"|^The web-page 'stop build' button was pressed"
         r"|.*\.js: line \d+, col \d+, Error -"
         r"|^\[taskcluster\] Error:"
+        r"|^\[[\w-]+:(?:error|exception)+\]"
```

Comment:
There's a stray `+` after the err/exception group, which will mean it matches against `[taskcluster:errorerror]`.

Could you also add a test to:
https://github.com/mozilla/treeherder/blob/f9fd2916847d71dd2f486e69830c014ad40bc3a3/tests/model/test_error_summary.py#L76
eg:
```python
    (
        '[taskcluster-vcs:error] Foo bar',
        '[taskcluster-vcs:error] Foo bar'
    ),
    (
        '[taskcluster:info] Foo bar',
        None
    ),
```

Thanks :-)